src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -84,7 +97,7 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
         <VerifiedSellerFragmentContainer artwork={artwork} />
         <BuyerGuaranteeFragmentContainer artwork={artwork} />
       </Join>
-      {!!shouldShowCreateAlertSection && (
+      {!!shouldShowCreateAlertSection && !hasEnded && (
```

Comment:
**quick suggestion**: move `!hasEnded` to [shouldShowCreateAlertSection](https://github.com/artsy/force/blob/main/src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx#L38) 

```javascript
const shouldShowCreateAlertSection = isCreateAlertButtonForArtworkEnabled && !artwork.is_sold && !hasEnded
```